packages/wallet/backend/src/rapyd/rapyd.d.ts

```diff
@@ -325,4 +325,24 @@ interface CompletePayoutRequest {
   amount: number
 }
 
+interface DailyRateRequest {
```

Comment:
Just fyi, this should become zod schemas, and the types being extracted from there. But I will add it in the rapy future proof merge request, it's fine.